scipy/stats/_rvs_sampling.py

```diff
@@ -0,0 +1,156 @@
+from __future__ import division, print_function, absolute_import
+
+import numpy as np
+import warnings
+from scipy._lib._util import check_random_state
+
+
+def rvs_ratio_uniforms(pdf, umax, vmin, vmax, size=1, c=0, random_state=None):
+    """
+    Generate random samples from a probability density function using the
+    ratio-of-uniforms method.
+
+    Parameters
+    ----------
+    pdf : callable
```

Comment:
What signature does pdf require?

Reply:
it is the pdf without further parameters